src/hooks/useSearch.ts

```diff
@@ -55,19 +55,20 @@ const useSearch = (query: TRouterQuery, familyId = "", documentId = "", runFresh
     }
 
     const cacheId = {
-      query_string: searchQuery.query_string,
+      concept_filters: searchQuery.concept_filters,
+      continuation_tokens: searchQuery.continuation_tokens,
+      corpus_import_ids: searchQuery.corpus_import_ids,
+      document_ids: searchQuery.document_ids,
       exact_match: searchQuery.exact_match,
+      family_ids: searchQuery.family_ids,
       keyword_filters: searchQuery.keyword_filters,
-      year_range: searchQuery.year_range,
+      metadata: searchQuery.metadata,
+      offset: searchQuery.offset,
+      query_string: searchQuery.query_string,
       sort_field: searchQuery.sort_field,
       sort_order: searchQuery.sort_order,
-      offset: searchQuery.offset,
-      family_ids: searchQuery.family_ids,
-      document_ids: searchQuery.document_ids,
-      continuation_tokens: searchQuery.continuation_tokens,
-      corpus_import_ids: searchQuery.corpus_import_ids,
-      concept_filters: searchQuery.concept_filters,
-      metadata: searchQuery.metadata,
+      sort_within_page: searchQuery.sort_within_page,
+      year_range: searchQuery.year_range,
```

Comment:
note: These reordering refactors make it a little tricky to see what the actual change is.

I've found it useful to do those refactors as no-ops first, and then introduce the change, but also understanding we're linting as we go.

Reply:
Apologies, I'm trying to find the right balance with this as I also don't particularly like the PR diff noise it creates.